compiler/ballerina-parser/src/test/resources/trivia/trivia_source_02.bal

```diff
@@ -1,3 +1,3 @@
 function foo() {
-     int a;
+    int a;
```

Comment:
Why we remove `NBSP` test?

Reply:
please refer to the https://github.com/ballerina-platform/ballerina-lang/issues/25147#issuecomment-787839199

Reply:
If this test failing because of a TODO, then let's keep the test as it is and update only the assert file. When we fix the TODO we can update the test. WDYT?

Reply:
+1